packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -84,6 +85,20 @@ const TradersHub = () => {
             >
                 {can_show_notify && <Notifications />}
                 <div id='traders-hub' className='traders-hub' ref={traders_hub_ref}>
+                    <WalletHeader account_type='demo' />
+                    <WalletHeader currency={'AUD'} jurisdiction={'svg'} />
```

Comment:
We can remove the curly braces if we just passing a string. Same goes to all the below 🙏

Reply:
@thisyahlen-deriv Agree, my fault again :)